src/terminal/node/terminal-backend-module.ts

```diff
@@ -22,13 +22,20 @@ export default new ContainerModule(bind => {
 @injectable()
 class TerminalExpressContribution implements BackendApplicationContribution {
     private terminals: Map<number, any> = new Map()
-    private logs: string[] = []
+
+    getShellExecutablePath() {
+        if (process.platform === 'win32') {
```

Comment:
It is in our code. Sorry for the confusion: https://github.com/theia-ide/theia/blob/master/src/application/common/os.ts#L8